front/src/common/api/osrdEditoastApi.ts

```diff
@@ -989,8 +985,7 @@ export type PostPathfindingApiArg = {
 };
 export type DeletePathfindingByIdApiResponse = unknown;
 export type DeletePathfindingByIdApiArg = {
-  /** Path ID */
-  id: number;
+  id?: number;
```

Comment:
Why this id is now optional ? I think it's a mistake, it should not become optional

Reply:
This is strange, I’m not sure what happened here. I’ll ask @leovalais

Reply:
Just a bad rebase :grimacing: 
It should be all right now